app/templates/components/billing/information.hbs

```diff
@@ -47,7 +47,7 @@
         </div>
         <div class='full-width'>
           <form.field
-            @label='Billing Email Address'
+            @label='Billing Email Addresses - Comma Separated'
```

Comment:
@emmanuel-ci thank you for the idea! I'm certainly in favor of multiple inputs instead of a comma separated list 👍 Also, we could implement something similar to what we had for "Migrate Beta" dialog:

![image](https://user-images.githubusercontent.com/4066489/78909270-f05eaa00-7a8b-11ea-9975-ef16be385cce.png)

(the code still exists in travis-web: https://github.com/travis-ci/travis-web/blob/master/app/components/dialogs/migrate-beta.js)

This is basically a select with multiple choices (which under the hood uses [Ember Power Select](https://ember-power-select.com/))

Although, it might be much harder to implement.

Reply:
I also like the idea of multiple inputs for this :+1:

Reply:
Great idea @artursmirnov 
moreover, we can use email validation in realtime here.

Reply:
@emmanuel-ci it's possible to create new items in power select on the fly. There's an add-on that enhances power select for this behavior: https://github.com/cibernox/ember-power-select-with-create, although I'm not sure if it's in a good condition and if it'd be feasible to integrate it. But you can also use it as a reference. 

If this way is too complicated, I'm also totally okay with the multiple boxes approach, as it seems to be quite common and intuitive as well 👍

Reply:
At this point I think I personally would lean towards the add more / multiple inputs approach. But the other approaches mentioned are good too :+1: